client/layout/guided-tours/positioning.js

```diff
@@ -110,25 +110,85 @@ function validatePlacement( placement, target ) {
 		: placement;
 }
 
-function scrollIntoView( target ) {
-	const targetSlug = target && target.dataset && target.dataset.tipTarget;
-
+export function getScrollDiff( targetSlug, container ) {
+	const target = targetForSlug( targetSlug );
+	const { top, bottom } = target.getBoundingClientRect();
 	if ( targetSlug !== 'themes' ) {
 		return 0;
 	}
-
-	const { top, bottom } = target.getBoundingClientRect();
-
 	if ( bottom + DIALOG_PADDING + DIALOG_HEIGHT <=
 			document.documentElement.clientHeight ) {
 		return 0;
 	}
-
-	const container = query( '#secondary .sidebar' )[ 0 ];
 	const scrollMax = container.scrollHeight -
 		container.clientHeight - container.scrollTop;
-	const y = Math.min( .75 * top, scrollMax );
 
-	scrollTo( { y, container } );
+	return Math.min( .75 * top, scrollMax );
+}
+
+function scrollIntoView( target ) {
+	const targetSlug = target && target.dataset && target.dataset.tipTarget;
+	if ( targetSlug !== 'themes' ) {
+		return 0;
+	}
+
+	const container = query( '#secondary .sidebar' )[ 0 ];
```

Comment:
- `container` is a bit generic here, since we're fetching a specific one: `sidebar` might be better?
- this pattern of `query`ing for the sidebar is repeated; should we define `getSidebarElement` or something?